packages/smart-wallet/src/smartWallet.js

```diff
@@ -747,6 +994,7 @@ export const prepareSmartWallet = (baggage, shared) => {
         const { helper } = facets;
 
         void helper.watchPurse(invitationPurse);
+        helper.repairUnwatchedSeats();
```

Comment:
please explain why this is called when a smart wallet is created. The timing of this is unclear to me. The sentinel in the contract's own baggage so it'll only ever be performed once per contract vat. But it doesn't happen until a new wallet is made. ???